rero_ils/modules/items/views.py

```diff
@@ -150,3 +150,12 @@ def item_availability_text(item):
                     number=item.number_of_requests(),
                     msg=request_txt)
         return text.strip()
+
+
+@blueprint.app_template_filter()
+def format_item_call_number(item):
+    """Returns formatted text to display call number for item."""
+    return '{call_number} - {second_call_number}'.format(
```

Comment:
Not very important:

`-` what if the first call number ends with `-`? will it be clear for user which one is the separator?

Reply:
Moreover, it could be nice to use `"-".join([call_number, second_call_number])`. It's more Pythonic I think :wink:

Reply:
done

Reply:
> Not very important:
> 
> `-` what if the first call number ends with `-`? will it be clear for user which one is the separator?

I modified separator to use `|`